src/api/clinical/diagnosis/diagnosis.auth.ts

```diff
@@ -39,7 +39,7 @@ export class DiagnosisAuth {
     static getById: AuthOptions = {
         ...DefaultAuthOptions,
```

Comment:
All the ownership should be with Owner. Scope will be tenant.

Reply:
Yes.